From a0943bd277ed9b5ecaf1c7c1ace7590b1b6da37d Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Mon, 20 Aug 2018 22:57:43 +0300
Subject: [PATCH] Remove inconsistent -fstrict-aliasing from ARM compiler configuration.

Test: Build arm devices and run hwui/skia performance tests.

diff --git a/cc/config/arm_device.go b/cc/config/arm_device.go
index 0d3750e..e89ea1a 100644
--- a/cc/config/arm_device.go
+++ b/cc/config/arm_device.go
@@ -114,7 +114,6 @@
 	armArmCflags = []string{
 		"-O2",
 		"-fomit-frame-pointer",
-		"-fstrict-aliasing",
 		"-funswitch-loops",
 	}
 
